Kuroba/app/src/main/java/com/github/adamantcheese/chan/core/cache/FileCacheDownloader.java

```diff
@@ -72,6 +73,10 @@ public FileCacheDownloader(Callback callback, String url, File output) {
         this.url = url;
         this.output = output;
 
+        okHttpClient = new OkHttpClient().newBuilder()
```

Comment:
You should use the dependency injected version of the class here, to not have a lot of clients being held; see https://square.github.io/okhttp/3.x/okhttp/okhttp3/OkHttpClient.html for details. I'll fix this when I merge it.

Reply:
Actually it might be better to just split a out a proxied OkHttpClient class to use as a singleton; I'll do that.